src/olympia/migrations/939-drop-l10n-from-bio-user-profile-cleanup.sql

```diff
@@ -0,0 +1,36 @@
+/* Follow-on from migration 938 -
+Drop the (now obsolete) bio and old translations. */
+
+DROP PROCEDURE IF EXISTS drop_bio_fk;
+
+DELIMITER ';;'
+
+CREATE PROCEDURE drop_bio_fk() BEGIN
+    /* drop the bio foreign key so we can clear the translations.
+    - we know prod and local dbs have different keys, so try to handle both */
+
+    /* prod fk name */
+    IF EXISTS (SELECT 1 FROM information_schema.table_constraints WHERE table_schema = (SELECT DATABASE()) AND table_name = 'users' AND constraint_name = 'users_ibfk_1') THEN
+        ALTER TABLE reviews DROP FOREIGN KEY users_ibfk_1;
+    END IF;
+
+    /* local docker default fk name */
+    IF EXISTS (SELECT 1 FROM information_schema.table_constraints WHERE table_schema = (SELECT DATABASE()) AND table_name = 'users' AND constraint_name = 'users_bio_21cc3d1ebf26e19_fk_translations_id') THEN
+        ALTER TABLE reviews DROP FOREIGN KEY users_bio_21cc3d1ebf26e19_fk_translations_id;
```

Comment:
Looks like `src/olympia/migrations/823-drop-backup-version.sql` is a better source of inspiration anyway. It allows us to retrieve the constraint name in one query which is much easier to read. So I'd replace the entire procedure with:

```sql
SET @CONSTRAINT_NAME := (SELECT CONSTRAINT_NAME FROM information_schema.key_column_usage WHERE TABLE_SCHEMA="olympia" AND TABLE_NAME="users" and COLUMN_NAME="bio" AND REFERENCED_TABLE_NAME = "translations");

SET @QUERY = CONCAT('ALTER TABLE users DROP FOREIGN KEY ', @constraint_name, ';');
PREPARE stmt FROM @QUERY;
EXECUTE stmt;
DEALLOCATE PREPARE stmt;
```